template.php

```diff
@@ -86,7 +87,14 @@ function uikit_base_preprocess_block(&$variables) {
 
   // Add some classes to the block title and content wrapper
   $variables['title_attributes_array']['class'] = 'block__title';
-  $variables['content_attributes_array']['class'] = 'block__content content';
+  $variables['content_attributes_array']['class'] = array('block__content', 'content');
+
+  // Global footer links class
+  if($variables['block']->region == 'footer_bottom'){
+    $variables['content_attributes_array']['class'][] = 'footer-links';
+  }
+
+  $variables['content_attributes_array']['class'] = join(' ', $variables['content_attributes_array']['class']);
 
```

Comment:
You don't need to collaspe the array, you can leave it as an array and Drupal will sort that for you, so you can remove this line.